applications/asset_tracker/src/ui/nmos.c

```diff
@@ -14,20 +14,18 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(ui_nmos, CONFIG_UI_LOG_LEVEL);
 
-#define NMOS_CONFIG_DEFAULT(_pin)	(.pin = _pin)
-#define NMOS_CONFIG_UNUSED		(.pin = -1)
-#define NMOS_CONFIG_PIN(_pin)		COND_CODE_0(_pin, NMOS_CONFIG_UNUSED,  \
-						NMOS_CONFIG_DEFAULT(_pin))
-#define NMOS_CONFIG(_pin)		{ NMOS_CONFIG_PIN(_pin),	       \
-						.mode = NMOS_MODE_GPIO }
-
+#define NMOS_CONFIG_DEFAULT(_pin) (.pin = _pin)
+#define NMOS_CONFIG_UNUSED (.pin = -1)
+#define NMOS_CONFIG_PIN(_pin)                                                  \
+	COND_CODE_0(_pin, NMOS_CONFIG_UNUSED, NMOS_CONFIG_DEFAULT(_pin))
+#define NMOS_CONFIG(_pin)                                                      \
+	{                                                                      \
+		NMOS_CONFIG_PIN(_pin), .mode = NMOS_MODE_GPIO                  \
+	}
 
 struct nmos_config {
 	int pin;
-	enum {
-		NMOS_MODE_GPIO,
-		NMOS_MODE_PWM
-	} mode;
+	enum { NMOS_MODE_GPIO, NMOS_MODE_PWM } mode;
```

Comment:
These (presumably machine-generated) patches are making the code less readable, so unless we are violating the coding convention they should be reverted. This applies elsewhere as well.